zebra/zebra_vrf.c

```diff
@@ -541,11 +576,22 @@ int zebra_vrf_netns_handler_create(struct vty *vty, struct vrf *vrf,
 	return CMD_SUCCESS;
 }
 
+static struct cmd_node vrf_node = {
```

Comment:
I used this because without I have zebra down during init with this message:
`mpls_fec_nexthop_resolution_vrf_cmd[[no$no] mpls fec nexthop-resolution]:
        node 21 does not exist.
        please call install_node() before install_element()
`
If you want to remove it you might have a comment how to overturn the consequence?

Reply:
I don't think redefining the structure here is the right answer though ... maybe @mjstapp might be able to help sort this?

Reply:
Hi @riw777, In this way we have it working. I agree that this not the best solution possible. If @mjstapp has a better way to suggest, I'll be happy to include it into the PR.

Reply:
try doing what bgpd does with vrf_cmd_init() ?